daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -404,40 +404,16 @@ func TestLogBlocking(t *testing.T) {
 	}
 }
 
-func TestLogNonBlockingBufferEmpty(t *testing.T) {
+func TestLogBufferEmpty(t *testing.T) {
```

Comment:
I'm not sure I understand the point of this test, but it should still pass with the non-blocking config removed, so I kept it. 

Running the tests locally now..